internal/commands/database/session/list.go

```diff
@@ -0,0 +1,187 @@
+package databasesession
+
+import (
+	"fmt"
+
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/commands"
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/completion"
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/format"
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/output"
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/resolver"
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/ui"
+
+	"github.com/UpCloudLtd/upcloud-go-api/v6/upcloud"
+	"github.com/UpCloudLtd/upcloud-go-api/v6/upcloud/request"
+	"github.com/spf13/pflag"
+)
+
+type listCommand struct {
+	*commands.BaseCommand
+	resolver.CachingDatabase
+	completion.Database
+	limit  int
+	offset int
+	order  string
+}
+
+// ListCommand creates the "session list" command
+func ListCommand() commands.Command {
+	return &listCommand{
+		BaseCommand: commands.New(
+			"list",
+			"List current sessions to specified database",
+			"upctl database session list 0fa980c4-0e4f-460b-9869-11b7bd62b832",
+			"upctl database session list mysql-1 --limit 16 --offset 32 --order pid:desc",
+		),
+	}
+}
+
+// InitCommand implements Command.InitCommand
+func (s *listCommand) InitCommand() {
+	flagSet := &pflag.FlagSet{}
+	flagSet.IntVar(&s.limit, "limit", 10, "Number of entries to receive at most.")
+	flagSet.IntVar(&s.limit, "offset", 0, "Offset for retrieved results based on sort order.")
+	flagSet.StringVar(&s.order, "order", "query_duration:desc", "Key and direction for sorting.")
+
+	s.AddFlags(flagSet)
+	_ = s.Cobra().MarkFlagRequired("pid")
```

Comment:
Is this here by accident, cannot see a `pid` flag above?

I wonder if the ignored error would be non nil in this case 🤔 We could think of some way to check those errors at some point

Reply:
Extra flag removed.